src/tests/FlapperMom.t.sol

```diff
@@ -49,71 +54,80 @@ contract FlapperMomTest is Test {
     event Stop();
 
     function setUp() public {
-        flapper = new FlapperUniV2(DAI_JOIN, SPOT, MKR, UNIV2_DAI_MKR_PAIR, PAUSE_PROXY);
-        assertLt(flapper.hop(), type(uint256).max);
+        FlapperInstance memory flapperInstance = FlapperDeploy.deployFlapperUniV2({
+            deployer: address(this),
+            owner:    PAUSE_PROXY,
+            daiJoin:  DAI_JOIN,
+            spotter:  SPOT,
+            gem:      MKR,
+            pair:     UNIV2_DAI_MKR_PAIR,
+            receiver: PAUSE_PROXY
+        });
+        flapper = FlapperUniV2(flapperInstance.flapper);
+        mom = FlapperMom(flapperInstance.mom);
+
+        FlapperUniV2Config memory cfg = FlapperUniV2Config(5 minutes, 1e18, address(0), 0); // use random values
```

Comment:
Nit: maybe better to use the full way to pass the struct so we can see the name of the params.

Reply:
Done (though it's all random values and the initFlapperUniV2() is called only to set the chief as the mom authority).